jboss/container/wildfly/launch/statefulset/added/launch/statefulset.sh

```diff
@@ -15,21 +15,25 @@ function configureEnv() {
 }
 
 function configure_socket_binding() {
-  init_node_name
+  if [ -z "$HOSTNAME" ]; then
+    1>&2 echo "Cannot define client mappings for socket binding. The remote EJB calls may not be working."
```

Comment:
I this is intended to be a warning, then you should use the log_warning function